recipes/nr.bundler/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nr.bundler" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f2217a0f1f2041ae3406d36a984ef14cc848aec6277a2b654235b7c40ea0cf5e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - appdirs >=1.4.3
+    - distlib >=0.2.7
+    - nr.fs >=1.3.1
+    - nr.types >=2.0.0
+    - pefile >=2017.11.5
+    - pip
+    - python
+    - setuptools
+  run:
+    - appdirs >=1.4.3
+    - distlib >=0.2.7
+    - nr.fs >=1.3.1
+    - nr.types >=2.0.0
+    - pefile >=2017.11.5
+    - python
+    - setuptools
```

Comment:
is setuptools needed as runtime dependency?

Reply:
Yes, setuptools is used in this library.